src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersStatusByEmailTest.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+import net.serenitybdd.junit.spring.integration.SpringIntegrationSerenityRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.professionalapi.controller.request.NewUserCreationRequest;
+
+@RunWith(SpringIntegrationSerenityRunner.class)
+@ActiveProfiles("functional")
+@Slf4j
+public class FindUsersStatusByEmailTest extends AuthorizationFunctionalTest {
+
+
+    @Test
+    public void ac1_find_user_status_by_email_with_pui_user_manager_role_should_return_200() {

```

Comment:
Please include your jira number and not only the ac number in the method signature i.e. rdcc672_ac1_find_user_status_by_email_with_pui_user_manager_role_should_return_200

Reply:
i think it is a good idea to put jira no in the functional test to build tracebility to the user stories. we should  try it out

Reply:
ok.